Samples/Controls/Selection/PaymentPage.ux

```diff
@@ -24,8 +24,8 @@
 				<Rectangle Layer="Background" Color="#DDD">
 					<Stroke Width="1" Color="#000"/>
 				</Rectangle>
-				<Selection Value="{Property this.Value}" MinCount="1" MaxCount="1"/>
-				<Each Items="{Property this.Items}">
+				<Selection Value="{ReadProperty this.Value}" MinCount="1" MaxCount="1"/>
```

Comment:
Just double-checking; did you test this works? It looks to me that `Selection` with one-way databinding might not be correct.